Source/JavaScriptCore/wasm/WasmOperationsInlines.h

```diff
@@ -58,8 +58,9 @@ JSWebAssemblyArray* fillArray(JSWebAssemblyInstance* instance, Wasm::FieldType f
     JSGlobalObject* globalObject = instance->globalObject();
     VM& vm = instance->vm();
 
-    auto* array = JSWebAssemblyArray::create(vm, globalObject->webAssemblyArrayStructure(), fieldType, size, WTFMove(rtt));
-    array->fill(vm, 0, static_cast<T>(value), size);
+    auto* array = JSWebAssemblyArray::tryCreate(vm, globalObject->webAssemblyArrayStructure(), fieldType, size, WTFMove(rtt));
```

Comment:
Should we rename `fillArray` to `tryFillArray`?

Reply:
Makes sense, thanks!